### PR TITLE
Category over mobile

### PR DESCRIPTION
**Pull request by phi-tran-vibe**

### CODE REVIEW

File: src/constants/index.js [hunk 1]
```diff
@@ -12,4 +12,5 @@ export const QUESTION_ANSWERED = 'QUESTION_ANSWERED';
 export const NEXT_CATEGORY = 'NEXT_CATEGORY';
 export const CORRECT_SYMBOL = 'O';
 export const INCORRECT_SYMBOL = 'X';
-export const ANSWER_CONTINUE = 'ANSWER_CONTINUE';
\ No newline at end of file
+export const ANSWER_CONTINUE = 'ANSWER_CONTINUE';
+export const MOBILE_WIDTH = 992;
```
Comment (phi-tran-vibe) [inline src/constants/index.js]: Adam could you confirm if this is the right width?

Comment (phi-tran-vibe) [reply src/constants/index.js]: ok, will change to 768 then

File: src/components/App.jsx [hunk 2]
```diff
@@ -16,49 +16,89 @@ import CategoryOver from '../containers/CategoryOver';
...
 class App extends React.Component {
+    state = {
+        width: window.innerWidth
+    };
+
+    componentWillMount() {
+        window.addEventListener('resize', _.debounce(this.onWindowSizeChange, 500));
+    }
+
+    componentWillUnmount() {
```
[the remark this answers is not in the document]
Comment (phi-tran-vibe) [reply src/components/App.jsx]: that's true, but it doesn't hurt to have it there i think (unbinding event listeners is a good habit)

[the remark this answers is not in the document]
Comment (adamrysyk) [reply src/components/App.jsx]: okay thanks

**Pull request merged**